docs/docs/FAQ.mdx

```diff
@@ -216,6 +216,9 @@ Scanning the entire video for faces may be implemented in the future.
 ### Does Immich have animal recognition?
 
 No.
+:::tip
+You can use [smart search](/docs/features/smart-search.md) for this to some extent. For example if you have a Golden Retriever and a Chihuahua, type these words in the smart search and watch the results.
```

Comment:
IMO when I see links like this ("Smart Search") I think they seems more like a proper term that should be capitalized.